platform/email/src/main/resources/webapp/js/services.js

```diff
@@ -6,7 +6,14 @@
     var services = angular.module('email.services', ['ngResource']);
 
     services.factory('SendEmailService', function($resource) {
-        return $resource('../email/send');
+        return $resource('../email/send',{}, {
+            save : {
+                method: 'POST',
+                transformResponse: function(response){
```

Comment:
yes, because before answer from server was putting in to function "fromJSON()" but the answer isn't a JSON and that why error was showed "SyntaxError: JSON.parse: unexpected character at line 1 colum 1 of the JSON data". Now we don't put the answer to this function.